### DISCUSSION TITLE
ブロックした相手のLGTMも消えるようになりませんか？あと、運営から削除されたイリーガルコメントも。

### DISCUSSION THREAD

**Discussion by fygar256** [increments/qiita-discussions#925] [要望・アイデア・不具合]
どうも僕はクラッカーに付き纏われているようで、ブロックしましたが、LGTMが残ります。
Qiitaにはブロック機能は必要はないかも、と思っていましたが（Qiitaはエンジニアの楽園を目指しているのかも）、
中にはとんでもない輩がいるようで、ブロック機能は仕方ないですね。そういうわけで、クラッカーをブロックしましたが、
LGTMが残ってしまいます。完全に抹消したいので、ブロックした相手のLGTMはカウントしないようになったらどうかと思います。
それだと、自分からは見えない自己満足ですが、目障りなものは遠ざけたい人間の心理。そういうわけで、違反通告して削除された
コメントの痕も消えるようにして欲しいです。かなり際どいお願いだと思いますが、ご考慮いただければと思います。
嫌われてるから、僕もやられるなあ。
色々すみません。

**Comment by tanaka-asami415** [comment]
@fygar256 
リクエストありがとうございます。
いただいた内容を整理させていただくと、以下のようなリクエストでお間違いないでしょうか？

1. Qiitaの[ブロック機能](https://help.qiita.com/ja/articles/qiita-block )をご活用いただいているものの、記事等につけられた「いいね」（[LGTMからいいねに戻りました](https://blog.qiita.com/released-lgtm-to-like/)）も表示したくない
1. 削除されたコメントを非表示にしたい

1 に関しては、『ご自身の記事やコメントに既につけられた「いいね」を無効にして欲しい』という事でしょうか？
それとも、ブロック後も『ブロックした相手に「いいね」を利用され、通知等で目にするのが嫌』という事でしょうか？

「いいね」は [contribution数](https://help.qiita.com/ja/articles/qiita-contribution) にも影響する事なので、どのような状態になるといいのかを詳しくお聞きした上で、検討させていただきたいと思います。

2 に関しては、社内で検討は行ったのですが、状況によっては全く何も無くなってしまうと文脈が分からない事もあるため、現在の仕様となっております。
一律に非表示にするというよりは、投稿者側でコメントの「表示」「非表示」を選択いただけるような改善は行いたいと考えております。

**Comment by fygar256** [comment]
ご返答ありがとうございます。
1に関しては、『ご自身の記事やコメントに既につけられた「いいね」を無効にして欲しい』ということです。
2に関しては了解しました。
なんか、クラッカーに、Qiitaで付き纏われているのは、僕の誤解だったようで、不快な思いをさせてしまいまして、
申し訳ありません。
でずが、万が一の場合を考えて、１、２のような機能はあったら良いかもと思います。

**Reply by tanaka-asami415** [reply]
@fygar256 
ご返信ありがとうございます！

> 1に関しては、『ご自身の記事やコメントに既につけられた「いいね」を無効にして欲しい』ということです。

承知しました。
まだ同様のご意見をいただいたことがないため、まずはどれぐらいニーズのある事なのか、Discussionsで他の方のご意見もお待ちしたいと思います🙇

2 の「コメントを非表示」にできるような機能は、他にもご要望としていただいているため、時期はお約束できませんが、改善を検討しております。

**Reply by fygar256** [reply]
どうもどうも。
